tables/gene_classification.tsv

```diff
@@ -0,0 +1,46 @@
+gene_type	gene_class	hg19	mm9
```

Comment:
good point. I will make sure documentation of the columns are clear in `scripts/visualize_genome_elements.py` in #18